doc/setup_server.rst

```diff
@@ -16,23 +16,18 @@ advise you to install it using the following command::
     conda install postgresql
 
 FIXME: I think that we can automatized this part.
-Once ``PostgreSQL`` install, we need to create a directory where to store the
-Postgres databases::
+Once ``PostgreSQL`` is installed, we initialize a Postgres database cluster::
 
-    mkdir postgres_dbs
```

Comment:
We should still need to create the folder, isn't it?

Reply:
Well, it was noted in #146 this was done by the initdb of postgres (but didn't test it)